.github/workflows/backstage-update.yml

```
@@ -0,0 +1,105 @@
+name: Update Backstage
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  schedule:
+    - cron:  '0 07 * * *'
+  workflow_dispatch:
+    
+jobs:
+  check-for-existing-update:
+    runs-on: ubuntu-latest
+    outputs:
+      exists: ${{ steps.pr.outputs.exists }}
+      url: ${{ steps.pr.outputs.url }}
+    steps:  
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Check for existing auto-update PR
+      id: pr
+      run: |
+        exists=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/department-of-veterans-affairs/lighthouse-backstage/pulls?head=department-of-veterans-affairs:auto-update-backstage)
+        url=$(echo $exists | jq .[] | jq .html_url)
+        echo ::set-output name=exists::$exists
+        echo ::set-output name=url::$url
+        
+  link-pr:
+    runs-on: ubuntu-latest
+    needs: check-for-existing-update
+    if: needs.check-for-existing-update.outputs.exists != '[ ]' 
+    steps:
+    - name: Failed to Update Backstage
+      uses: actions/github-script@v3
+      with:
+        script: |
+          console.log("Failed to Update Backstage - Pull Request already exists: ",${{ needs.check-for-existing-update.outputs.url }})
+      
+ 
+  update-backstage:
+    runs-on: ubuntu-latest
+    needs: check-for-existing-update
+    if: needs.check-for-existing-update.outputs.exists == '[ ]' 
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install Dependencies
+        run: |
+          yarn install
+      - name: Update Backstage
+        run: |
+          npm run backstage-update
+      - name: Compare Create App versions
+        id: compare_versions
+        run: |
+          mostRecent=$(npx @backstage/create-app --skip-install --version)
+          current=$(cat package.json | jq .metadata | jq '."backstage-create-app-version"' | sed 's/"//g')
+          echo ::set-output name=current::$current
+          echo ::set-output name=mostRecent::$mostRecent
+      - name: Format PR body
+        id: get-pr-body
+        run: |
+          body=$(sed -n '/breaking changes/, $p' backstage-update-log.txt)
+          body="${body//$'\n'/'%0A'}"
+          echo ::set-output name=body::$body
+      - name: Set output variables
+        id: vars
+        run: |
+          echo ::set-output name=pr_title::"Update Backstage (auto-generated)"
+          echo ::set-output name=date::$(date +'%Y-%m-%d')
+      - name: Create Pull Request
+        id: cpr
```

```suggestion
        id: create-pr
```
This doesn't have to be overly terse. Feel free to call it something else.